src/darwin/Framework/CHIP/MTRCommissioneeInfo.h

```diff
@@ -48,6 +48,13 @@ MTR_AVAILABLE(ios(18.4), macos(15.4), watchos(11.4), tvos(18.4))
  */
 @property (nonatomic, copy, readonly, nullable) MTREndpointInfo * rootEndpoint;
 
+/**
+ * A dictionary mapping endpoint IDs to FeatureMap values for the Network
+ * Commissioning cluster, for endpoints that implement that cluster.  May
+ * contain no entries if there are no such endpoints.
+ */
+@property (nonatomic, copy, readonly) NSDictionary<NSNumber *, NSNumber *> * networkCommissioningFeatureMaps MTR_AVAILABLE(ios(26.1), macos(26.1), watchos(26.1), tvos(26.1));
```

Comment:
An NSNumber to NSNumber dictionary doesn't seem like the greatest API... could we have e.g. `wiFiInterface`, `threadInterface`, and `ethernetInterface` properties? I guess we'd have to introduce something like `MTRNetworkInterfaceInfo` that has the endpoint number and the feature map on it.

Reply:
Per Slack discussion: if we do MTRNetworkInterfaceInfo we can use it in more places later.

Reply:
Updated the API.